src/main/java/com/openlattice/shuttle/transforms/ColumnTransform.java

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
This is a great time to turn some of the simpler classes that were moved into Kotlin classes :D